src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private final Random random = new Random();
+
+    public Random getRandom() {
+        return random;
+    }
+
+    public String getRandomBall() {
+
+        int maxPossibleNumber = 100;
```

Comment:
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-name-for-getting-string-representation-of-enum-constants

Reply:
Please see [this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names) to learn about constant fields and their naming requirements.